### DISCUSSION TITLE
Installation: Creating Wiki Docker Container Fails

### DISCUSSION THREAD

**Discussion by EORG-Dev** [requarks/wiki#6828] [Error / Bug Report] [answered]
I'm trying to install wiki.js on my ubuntu 22.04 LTS server hosted by hetzner. It's a clean and uptodate install of ubuntu running on the CX11 (vCPU, x86) plan. 

I'm following this guide: https://docs.requarks.io/install/ubuntu
All steps up to "Setup Containers" -> "docker create --name=wiki [...]" execute without Issues.

When executing: "docker create --name=wiki -e DB_TYPE=postgres -e DB_HOST=db -e DB_PORT=5432 -e DB_PASS_FILE=/etc/wiki/.db-secret -v /etc/wiki/.db-secret:/etc/wiki/.db-secret:ro -e DB_USER=wiki -e DB_NAME=wiki -e UPGRADE_COMPANION=1 --restart=unless-stopped -h wiki --network=wikinet -p 80:3000 -p 443:3443 ghcr.io/requarks/wiki:2"

Expected behavior: no issues raised, missing data is pulled from the internet.

Actuall behavior/output: 
"Unable to find image 'ghcr.io/requarks/wiki:2' locally
Error response from daemon: Get "https://ghcr.io/v2/": dial tcp 140.82.121.34:443: i/o timeout (Client.Timeout exceeded while awaiting headers)"

I've tried: 
- Checking github availability 
- pinging ghcr.io -> 100% package loss, no outbound firewall 
- Setting up a new and clean server.
- Researching whether this is a known or acute issue

Background: I'm successfully operating 3 other CX11 Hetzner servers without any IP/DNS/Connection issues, however I'm not formaly educated in this bussiness and focused on mobile device software development.

I would appreachiate any help.

**Comment by NGPixel** [comment, marked as answer]
Try with the image from docker hub? Just `requarks/wiki:2`

Closing this issue as this is a network issue between your server and the GitHub container registry.
